### PR TITLE
add NoChangePassword to userDetail check

### PR DESCRIPTION
**Pull request by Mobmaker55**
As with issue #133, the NoChangePassword was never implemented to userDetail. This commit changes that.

### CODE REVIEW

**Comment by Mobmaker55** [comment]
I realize that first commit is in there now that I'm looking at it... I have no idea how to remove it, but it doesn't *break* anything, so can it stay? or do I have to remove it

File: cmd/checks_windows.go [hunk 1]
```diff
@@ -353,6 +353,8 @@ func userDetail(userName, detailName, detailValue string) (bool, error) {
 		return user.IsAdmin == lookingFor, nil
 	case "PasswordNeverExpires":
 		return user.PasswordNeverExpires == lookingFor, nil
+	case "NoChangePassword:
```
Comment (sourque) [inline cmd/checks_windows.go]: Missing a closing quote :(

**Comment by sourque** [comment]
> I have no idea how to remove it, but it doesn't _break_ anything, so can it stay? or do I have to remove it

It's probably best if you remove it, to help keep things organized. You can use `git rebase` to help out ([resource](https://git-rebase.io/)).

File: cmd/checks_windows.go (hunk 1 above)
Comment (Mobmaker55) [reply cmd/checks_windows.go]: it somehow managed to compile without the closing quote, I have no idea how that happened

**Comment by safinsingh** [comment]
Can you merge master into your commit?

**Comment by safinsingh** [comment]
Bump^

**Comment by Mobmaker55** [comment]
@safinsingh I should have it fixed soon -- might need to make a new PR though bc it's no longer linked to the branch I had previously :/

**Comment by Mobmaker55** [comment]
Looking into it, there are currently 5 (including NoChangePassword) attributes to users that we are not currently scoring... is it a good idea to add the rest as well ? (PasswordAge, LastLogon, BadPasswordCount, NumberOfLogons)
Most of the others are probably niche but it allows more configuration without going to commands...

**Comment by Mobmaker55** [comment]
#145 solves this PR, stupid GH not seeing my old branch but oh well.

**Pull request closed without merging**